src/libs/ReportUtils.js

```diff
@@ -1073,15 +1073,14 @@ function getIOUReportActionMessage(type, total, participants, comment, currency,
  * @param {String} currency
  * @param {String} comment - User comment for the IOU.
  * @param {Array}  participants - An array with participants details.
+ * @param {String} iouTransactionID - Only required if the IOUReportAction type is oneOf(cancel, decline). Generates a randomID as default.
```

Comment:
Let's remove the iou terminology here since we're trying to move away from that.

```suggestion
 * @param {String} TransactionID - Only required if the IOUReportAction type is oneOf(cancel, decline). Generates a randomID as default.
```

Reply:
transactionID

Reply:
Ok, I can change the parameter name with only a trivial amount of effort. Do we want to do just that, or do we want to migrate all IOU reportActions `originalMessage.IOUTransactionID` -> `originalMessage.transactionID`?

Because we can do that, but it's a more significant migration and should probably have its own issue if we think it's worth it.

Reply:
Updated the parameter name, if you want to rename the actual field let's create another issue for the migration.